core/src/main/resources/lib/form/hetero-list.jelly

```diff
@@ -82,31 +82,26 @@ THE SOFTWARE.
     <d:tag name="body">
       <div style="width:100%">
         <j:set var="help" value="${descriptor.helpFile}" />
-        <j:if test="${hasHeader}">
-          <div class="tr help-sibling">
-            <div colspan="3">
-              <div class="${attrs.disableDragAndDrop or readOnlyMode ? '' : 'dd-handle'}">
-                <b>${descriptor.displayName}</b>
-              </div>
-              <f:helpLink url="${help}"/>
-            </div>
-          </div>
-          <!-- TODO: help support is unintuitive; people should be able to see help from drop-down menu -->
-          <j:if test="${help!=null}">
-            <f:helpArea />
+
+        <div class="repeated-chunk__header">
+          <div class="${attrs.disableDragAndDrop or readOnlyMode ? '' : 'dd-handle'}"/>
+
+          ${descriptor.displayName}
+
+          <f:helpLink url="${help}"/>
+
+          <j:if test="${!readOnlyMode}">
+            <f:repeatableDeleteButton value="${attrs.deleteCaption}" />
           </j:if>
-        </j:if>
+        </div>
 
-        <d:invokeBody/>
+        <j:if test="${help!=null}">
```

Comment:
something in this or entry I think has broken repeatable elements from selecting the right text area for help area.

Steps:
1. Create a freestyle job
2. Select `This project is parameterised`
3. Add a Boolean parameter
4. Add a File parameter
5. Click the help icon next to file parameter

Expect it to be outputted in the section for the file parameter
Actually the help box appears in the boolean parameter section

![image](https://user-images.githubusercontent.com/21194782/144013009-9c7b6b78-d156-4470-99c4-83c69f0a1c87.png)

Reply:
Great spot (and step list!) - fixed :) 

<img width="683" alt="image" src="https://user-images.githubusercontent.com/43062514/144034473-5ae75148-9afe-4232-84d6-0786e9c52020.png">

Reply:
sorted